### DISCUSSION TITLE
`breakParent` refuses to break parent group

### DISCUSSION THREAD

**Discussion by mattfysh** [prettier/prettier#14709] [Q&A] [answered]
I'd like to create a group that breaks if any children contain `breakParent`, but the following code is not working. The output I'd expect to see because of the breakParent is:

```
{
  a
  b
  c
  d
}
```

but instead what I see is: `{ a, b, c, d }` -- but I would only expect to see this is the `entries.push...` line was commented out

```js
const { doc } = require('prettier')

const { group, line, indent, join, ifBreak, hardline, breakParent } =
doc.builders;

const sep = ifBreak(hardline, [',', line])
const entries = ['a', 'b', 'c']
entries.push(['d', breakParent])
const x = group([
  '{',
  indent([
    line,
    join(sep, entries)
  ]),
  line,
  '}'
])

console.log(
doc.printer.printDocToString(x, {
  printWidth: 200,
  tabWidth: 2,
  useTabs: false,
}).formatted
);
``

**Comment by mattfysh** [comment]
I think this may be fixed by https://github.com/prettier/prettier/pull/14466/files - does that sound right @fisker?

**Reply by fisker** [reply, marked as answer]
Yes, possible. We should add test for it.
